aeneid-testnet/hook-modules.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "name": "LockLicenseHook",
+    "address": "0x54C52990dA304643E7412a3e13d8E8923cD5bfF2",
+    "blockExplorerLink": "https://aeneid.storyscan.io/address/0x54C52990dA304643E7412a3e13d8E8923cD5bfF2?tab=contract",
+    "whitelisted": true
+  },
+  {
+    "name": "TotalLicenseTokenLimitHook",
+    "address": "0xaBAD364Bfa41230272b08f171E0Ca939bD600478",
+    "blockExplorerLink": "https://aeneid.storyscan.io/address/0xaBAD364Bfa41230272b08f171E0Ca939bD600478?tab=contract",
+    "whitelisted": true
+  },
+  {
+    "name": "IPUSDPriceHook",
+    "address": "0x14dd2b86137ef09806980fbeda4cd11cfc466d59",
```

Comment:
Took a first pass at the contract - sharing questions/comments:

1. @kingster-will `bytes32 key = keccak256(abi.encodePacked(licensorIpId, licenseTemplate, licenseTermsId))` - can the same key be used for multiple IPs? If yes, this could be an issue for the IP owner.
2. `function updateIpPrice(bytes[] calldata pythPriceUpdate) external payable` - why is this function needed if on `_calculateFee` we get latest price?
3. What is the correct way to get latest price? is it `getPriceNoOlderThan` or is there a better one?
4. IP owner can frontrun txs and alter price right before the buyer’s payment. A cooldown on price change could resolve this.